src/ontology/mondo-edit.obo

```diff
@@ -434186,7 +434186,9 @@ xref: ICD10:U07.1 {source="MONDO:equivalentTo"}
 xref: ICD10:U07.2 {source="MONDO:relatedTo"}
 xref: ICD11:RA01.0 {source="MONDO:equivalentTo"}
 xref: ICD11:RA01.2 {source="MONDO:relatedTo"}
+xref: MESH:C000657245
 xref: SCTID:840539006 {source="MONDO:equivalentTo"}
+xref: SNOMEDCT:840539006
```

Comment:
What do you call the extra information in `{}`? Is it okay that my additions don't have it?

Reply:
It is the axiom annotation on the xref, you can read more about it here:
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#axiom-annotations-on-xrefs

The prefix  for SNOMED should be SCTID (SNOMED Clinical Terminology ID). Would you mind fixing that? Thanks!

Reply:
> The prefix for SNOMED should be SCTID

Will change.

I assumed the preferred prefix would follow identifiers.org, [which uses](https://registry.identifiers.org/registry/snomedct) `snomedct`. I know in some cases identifiers.org is wrong, for example it doesn't separate icd9, icd10, and icd11 and instead just has a prefix for icd. Do you think identifiers.org made the wrong choice here? It seems to me like `sctid` is the appropriate name for identifiers from SNOMED-CT, but `snomedct` might be a more appropriate prefix to refer to the resource (as opposed to a specific identifier).

Reply:
Good question - I am not sure how the decision was made to use SCTID instead of SNOMEDCT. @cmungall could you please comment? Thanks!

Reply:
> It is the axiom annotation on the xref

Good to know. So should I add any axiom annotations or are the xrefs with annotations imported from external ontologies, and the axioms reflect the nature of the import?

Reply:
Actually, yes- could you add MONDO:equivalentTo to the MESH xref. We don't add equivalence axioms to SNOMED though. Thanks!

Reply:
Dont worry about the word denoting the namespace too much (SNOMEDCT vs SCTID). What matters is how this is unfolded to IRIs, which Mondo does not do (for SNOMED). My guess is that SCTID was picked because that's what SNOMED people call it: https://confluence.ihtsdotools.org/display/DOCGLOSS/SCTID

Reply:
> Dont worry about the word denoting the namespace too much (SNOMEDCT vs SCTID). What matters is how this is unfolded to IRIs, which Mondo does not do (for SNOMED).

Is there a canonical method for unfolding xrefs to IRIs that handles poorly standardized xref prefixes? In my case, I just want the xrefs in a standardized `prefix:accession` format (and not as IRIs).

This is sort of off topic to this PR (and perhaps even this repo), so feel free to suggest an alternative location if preferable or we envision continued discussion.